src/DynamoPublish/DynamoPublish.csproj

```diff
@@ -30,36 +32,114 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
+    <Reference Include="Greg, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\extern\greg\Greg.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
+    <Reference Include="GregStandaloneAuth, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
GregStandaloneAuth should not be included with your pull request.  So, this reference is unnecessary.